packages/schematics/src/collection/ng-new/index.ts

```diff
@@ -1,73 +1,160 @@
-import {
-  apply,
-  branchAndMerge,
-  chain,
-  mergeWith,
-  Rule,
-  SchematicContext,
-  template,
-  Tree,
-  url
-} from '@angular-devkit/schematics';
+import { chain, move, Rule, schematic, Tree } from '@angular-devkit/schematics';
 import { Schema } from './schema';
-import { strings } from '@angular-devkit/core';
-import {
-  NodePackageInstallTask,
-  RepositoryInitializerTask
-} from '@angular-devkit/schematics/tasks';
-import { libVersions } from '../../lib-versions';
-import { DEFAULT_NRWL_PRETTIER_CONFIG } from '../../utils/common';
+import { addImportToModule, insert } from '../../utils/ast-utils';
+import * as ts from 'typescript';
+import { insertImport } from '@schematics/angular/utility/ast-utils';
 
 export default function(options: Schema): Rule {
-  if (!options.name) {
-    throw new Error(`Invalid options, "name" is required.`);
-  }
   if (!options.directory) {
     options.directory = options.name;
   }
 
-  return (host: Tree, context: SchematicContext) => {
-    addTasks(options, context);
-    const npmScope = options.npmScope ? options.npmScope : options.name;
-    const templateSource = apply(url('./files'), [
-      template({
-        utils: strings,
-        dot: '.',
-        tmpl: '',
-        ...libVersions,
-        ...(options as object),
-        npmScope,
-        defaultNrwlPrettierConfig: JSON.stringify(
-          DEFAULT_NRWL_PRETTIER_CONFIG,
-          null,
-          2
-        )
-      })
+  const opts = { ...options, preset: undefined };
+
+  return chain([
+    schematic('workspace', opts),
```

Comment:
This should `filter` out what was originally in the directory

```suggestion
    filter(() => false),
    schematic('workspace', opts),
```